diff -ur a/system.sh b/system.sh
--- a/system.sh	2023-10-01 15:18:48.616765777 -0400
+++ b/system.sh	2023-10-01 15:21:32.864677803 -0400
@@ -252,8 +252,10 @@
                     error="You need Linux Mint Debian Edition 4 or newer"
                 elif compareVersions "$__os_release" lt 5; then
                     __os_debian_ver="10"
-                else
+                elif compareVersions "$__os_release" lt 6; then
                     __os_debian_ver="11"
+                else
+                    __os_debian_ver="12"
                 fi
             fi
             ;;
